submissions/chabuChabu/phpDatabase/query.php

```diff
@@ -0,0 +1,17 @@
+<?php
+// 3. there is no cookie or matching id, load data from database
+$sql = "SELECT customers.id,products.product_name,order_items.unit_price,order_items.quantity,order_items.total_amount,
```

Comment:
The whole thing you did here should be in a function

Reply:
@adaugochi ok noted